ts/modules/series-label.src.ts

```diff
@@ -1127,12 +1144,17 @@ function drawLabels(this: Highcharts.Chart, e: Event): void {
 
             // The labels are processing heavy, wait until the animation is done
             if (e.type === 'load') {
-                delay = Math.max(
+                delay = Math.min(
                     delay as any,
                     animObject(series.options.animation).duration as any
                 );
             }
-
+            // Add the label or the series defer time
+            if (options.defer || defer) {
+                isNumber(options.defer) ?
+                    (delay as any) += options.defer :
+                    delay += defer;
+            }
```

Comment:
It took me a while - something doesn't feel right here. The docs say: 
```
                                              ...Setting to `false` renders
                 * the series label immediately. If set to `true` inherits
                 * the defer time set in [series.animation](#series.animation).
```
If `options.defer` is `true` then `delay += defer` runs no matter what the value of `defer` is. I'm ok with that.
If `options.defer` is `false` then `delay += defer` runs if `defer` is not zero (as it has to be a number), so this makes me think that something is wrong here.
And if `options.defer` is zero then `delay += options.defer` runs (if `delay` is not zero) or nothing happens, so basically the outcome is the same - no change to `delay`. This 3rd scenario fits better the description of `renders the series label immediately`.